### PR TITLE
Configure uptime only when the monitoring package is purchased

### PR DESCRIPTION
**Pull request by Kjir**

### CODE REVIEW

File: ruby_on_rails/README.md [hunk 1]
```diff
@@ -31,9 +31,12 @@ They are not optional, but the rest of the team can start working even if those
 
 1. [GetSentry](getsentry.md)
 1. [NewRelic](newrelic.md)
-1. [Uptime Monitor](uptime.md)
 1. [Gemnasium](gemnasium.md)
 
+Some services should be enabled only if the customer pays for the monitoring package:
```
Comment (Kjir) [inline ruby_on_rails/README.md]: Should New Relic also come under this section? What other services belong here?

Comment (coorasse) [reply ruby_on_rails/README.md]: Gemnasium, NewRelic, Sentry as well

File: ruby_on_rails/uptime.md [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
-if we would do this for all environments.
+To ensure that our application is always up and running, we offer a monitoring
+package to the customers. Those who decide to apply for this package will get
+the added benefit of a monitored application, so that we can act quickly if the
+application goes down.
+
+This check should be enabled only when the following conditions are met:
+  * The customer applied for the monitoring package
+  * The application has a paid dyno
+  * The application is live, or the go-live is imminent.
```
Comment (coorasse) [inline ruby_on_rails/uptime.md]: why is this point relevant? Aren't the other two enough?

File: ruby_on_rails/README.md [hunk 3]
```diff
@@ -32,7 +32,10 @@ They are not optional, but the rest of the team can start working even if those
 
 1. [GetSentry](getsentry.md)
 1. [NewRelic](newrelic.md)
-1. [Uptime Monitor](uptime.md)
+
+Some services should be enabled only if the customer pays for the monitoring package:
+
+1. [Uptime](uptime.md)
```
Comment (macav) [inline ruby_on_rails/README.md]: can we link here the document with the monitoring overview? https://docs.google.com/spreadsheets/d/1FY4jqByO-aI5sDan0hD7ULu6a2-eLsmO6kgdCFlPmuY/edit#gid=0

if there is uptime SMS, then the "high priority" contact should be chosen

also based on the document we should enable depfu and set the sentry monitoring team

**Comment by macav** [comment]
@coorasse Ci si failing

**Pull request merged**